recipes/dvc-task/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "dvc-task" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/dvc-task-{{ version }}.tar.gz
+  sha256: 9f5a9fe9c44791c6f87d9bf182115a2a59faf660971c630ae4c7dd16327296e2
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+    - setuptools_scm
+  run:
+    - celery
+    - funcy
+    - kombu
+    - python >=3.7
+    - pywin32-on-windows
+    - shortuuid
```

Let's match the version ranges from upstream:
```suggestion
    - celery >=5.2.0,<6
    - funcy >=1.17
    - kombu
    - python >=3.7
    - pywin32-on-windows >=225
    - shortuuid >=1.0.8
```

---

This is done (with the note that `pywin32-on-windows` is still unversioned and the `pywin32>=225` constraint is in `run_constrained`)